proactive/ProactiveGateway.py

```diff
@@ -446,11 +450,11 @@ def getAllJobs(self, max_number_of_jobs=1000):
         :param max_number_of_jobs: The maximal number of retrieved jobs
         :return: A list of jobs
         """
-        job_filter_criteria = self.runtime_gateway.jvm.org.ow2.proactive.scheduler.common.JobFilterCriteria(False, False,
-                                                                                                            True, False,
+        job_filter_criteria = self.runtime_gateway.jvm.org.ow2.proactive.scheduler.common.JobFilterCriteria(False, True,
```

Comment:
I would prefer if we improve this method by making it configurable. We can then keep the default of listing only running jobs

Reply:
We could add another one that is configurable and keep the listing all jobs as it is.

Reply:
I have the feeling this was set on running jobs only for a reason (I know the python client is used by customers) so it could be a request coming from a customer. Adding extra parameters with default values will not break backward compatibility, no ?

Reply:
I see your point. And you're right, adding extra parameters with default values will not break backward compatibility. I will do the change.